### PR TITLE
Feature/swepub oa

### PR DESCRIPTION
**Pull request by klngwll**

### CODE REVIEW

**Review by olovy — APPROVED** [review]
LGTM!

File: source/swepub/terms.ttl [hunk 1]
```diff
@@ -69,3 +69,8 @@ swepub:EpubAheadOfPrintOnlineFirst a swepub:PublicationStatus ;
     skos:prefLabel "Icke-sakkunninggranskad publiceringskanal"@sv ;
     rdfs:comment "Kanalen uppfyller inte kriterierna för sakkunniggranskad publiceringskanal."@sv ;
     rdfs:comment "Publication channel that does not meet the criteria for a peer-reviewed publication channel."@en .
+
+</term/swepub/doaj-journal> a :Status ;
```
Comment (niklasl) [inline source/swepub/terms.ttl:73]:
```suggestion
</term/swepub/doaj-journal> a swepub:PublicationStatus ;
```
?

Comment (klngwll) [reply source/swepub/terms.ttl]: The status refers to the DOAJ-indexing.

**Review by niklasl — APPROVED** [review]
LGTM

:eyes: on the TODO (would like to do that; given stability in time of the entities being subject of a status).

**Pull request merged**